src/otg/datasource/gwas_catalog/study_index.py

```diff
@@ -117,53 +92,6 @@ def from_source(
             ._annotate_discovery_sample_sizes()
         )
 
-    def get_gnomad_population_structure(self: GWASCatalogStudyIndex) -> DataFrame:
```

Comment:
The `df.transform(fun)` approach got dropped. Now this operation is column based.